src/index.ts

```
@@ -3,4 +3,6 @@ export * from './components/Button/Button';
 export * from './components/IconRoundedButton/IconRoundedButton';
 export * from './components/SelectionButton/SelectionButton';
 export * from './components/Text/Text';
-export * from './components/Table/Table';
+export * from './components/Toast/Toast';
+export * from './components/Toast/ToastProvider';
+export * from './components/Table/Table';
```

ESLint

[error] Insert `⏎`

(prettier/prettier)